anomalib/deploy/inferencers/torch_inferencer.py

```diff
@@ -84,7 +104,7 @@ def load_model(self, path: Union[str, Path]) -> AnomalyModule:
         model = get_model(self.config)
         model.load_state_dict(torch.load(path)["state_dict"])
```

Comment:
Don't we need to pass the map_location here? It might fail otherwise if we load a gpu model on cpu.
https://pytorch.org/tutorials/recipes/recipes/save_load_across_devices.html

Reply:
Good point